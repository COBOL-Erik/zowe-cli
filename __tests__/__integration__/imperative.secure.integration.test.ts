/*
* This program and the accompanying materials are made available under the terms of the
* Eclipse Public License v2.0 which accompanies this distribution, and is available at
* https://www.eclipse.org/legal/epl-v20.html
*
* SPDX-License-Identifier: EPL-2.0
*
* Copyright Contributors to the Zowe Project.
*
*/

// These tests require access to the same values on the keyring, therefore they cannot run in parallel
// The test order is important - some tests depend on other tests not running first - do not change it
// All tests that mess with the keyring must go here - the MacOS Keyring is NOT thread safe, and cannot run anything in parallel
/* eslint-disable max-len */

describe("Imperative Secure Tests", () => {
    require("./../../packages/imperative/__tests__/src/packages/imperative/__integration__/ConfigLoading.integration.subtest");
    require("./../../packages/imperative/__tests__/src/packages/imperative/__integration__/PluginManagementFacility.integration.subtest");
    require("../../packages/imperative/__tests__/__integration__/imperative/__tests__/__integration__/cli/cli.imperative-test-cli.integration.subtest");
    require("../../packages/imperative/__tests__/__integration__/cmd/__tests__/integration/cli/auth/Cmd.cli.auth.fruit.integration.subtest");
});
