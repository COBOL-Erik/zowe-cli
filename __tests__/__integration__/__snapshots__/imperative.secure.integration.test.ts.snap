// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Imperative Secure Tests Imperative Test CLI Secure Tests imperative-test-cli config import failure scenarios should fail to import if location is not specified 1`] = `
"
Syntax Error:
Missing Positional Argument: location
Argument Description: File path or URL to import from.

Example:

 - Import config from local file on disk:

      $ imperative-test-cli config import ~/Downloads/zowe.config.json

Use \\"imperative-test-cli config import --help\\" to view command description, usage, and options.
"
`;

exports[`Imperative Secure Tests Imperative Test CLI Secure Tests imperative-test-cli config list should list the configuration 1`] = `""`;

exports[`Imperative Secure Tests Imperative Test CLI Secure Tests imperative-test-cli config list should list the configuration without showing secure values 1`] = `""`;

exports[`Imperative Secure Tests Imperative Test CLI Secure Tests imperative-test-cli config list should list the defaults configuration property 1`] = `""`;

exports[`Imperative Secure Tests Imperative Test CLI Secure Tests imperative-test-cli config list should list the profiles configuration property 1`] = `""`;

exports[`Imperative Secure Tests Imperative Test CLI Secure Tests imperative-test-cli config list should list the root level property names only 1 1`] = `""`;

exports[`Imperative Secure Tests Imperative Test CLI Secure Tests imperative-test-cli config list should list the root level property names only 2 1`] = `""`;

exports[`Imperative Secure Tests Imperative Test CLI Secure Tests imperative-test-cli config profiles should list profiles 1`] = `""`;
